app/views/components/_error-summary.html.erb

```diff
@@ -0,0 +1,31 @@
+<%
+  description ||= false
+  items ||= []
+  title_id ||= "error-summary-title-#{SecureRandom.hex(4)}"
+%>
+<div
+  class="app-c-error-summary"
+  data-module="error-summary"
+  aria-labelledby="<%= title_id %>"
+  role="alert"
+  tabindex="-1"
+>
+  <h2 class="app-c-error-summary__title" id="<%= title_id %>">
+    <%= title %>
+  </h2>
+  <div class="app-c-error-summary__body">
+    <% if description %>
+      <p class="app-c-error-summary__text"><%= description %></p>
+    <% end %>
+    <% items.each_with_index do |item, index| %>
+      <ul class="app-c-error-summary__list">
```

Comment:
I think this line needs to be outside the loop - in the 2nd example in the component guide you're outputting three links, each wrapped in their own list.

Reply:
Good catch thanks Andy